specification/apimanagement/resource-manager/readme.md

```diff
@@ -1059,7 +1059,13 @@ directive:
     reason: Warning raised to error while PR was being reviewed. SystemData will implement in next preview version.  
   - suppress: R4009
     from: apimpolicyfragments.json
-    reason: Warning raised to error while PR was being reviewed. SystemData will implement in next preview version. 
+    reason: Warning raised to error while PR was being reviewed. SystemData will implement in next preview version.
```

Comment:
Please close this PR and suppress the errors with the changes together. But only errors caused by existing content may be able to suppress. Errors caused by new content must be fixed.